scripts/install.sh

```diff
@@ -17,6 +17,9 @@ if [[ ${DEBUG:-false} == "true" ]]; then
 fi
 
 export PKG_KREW_PLUGINS_LIST=" "
+export PKG_KIND_VERSION=0.17.0
```

Comment:
Where is PKG_KIND_VERSION being used?

Reply:
That is used [here](https://github.com/electrocucaracha/pkg-mgr_scripts/blob/master/scripts/kind/main.sh#L39), the external script called to install dependencies.

Reply:
@electrocucaracha appreciate the response.

FYI : When @sandeepAarna and I were testing his deployment of KIND clusters for free5gc, we had to execute the below in order for clusters with CNI and other plugins to be installed. 

```
echo fs.inotify.max_user_watches=655360 | sudo tee -a /etc/sysctl.conf
echo fs.inotify.max_user_instances=1280 | sudo tee -a /etc/sysctl.conf
```
What are your thoughts on where the above should be added?

Suggestion (feel free to disregard): Is this now tested on one of the GCP VMs that we all now have access to ensure that the script is working as desired? It would make merging the PR easy.

Reply:
Sure, I can include those instructions in this PR